snapcraft/_store.py

```diff
@@ -73,6 +75,84 @@ def logout():
     logger.info('Credentials cleared.')
 
 
+def _get_usable_secret_keys(query=None):
+    ctx = gpgme.Context()
```

Comment:
we usually try to avoid variable names like this. Specially for somebody like me who doesn't speak great English, ctx is less readable than context.